docs/guides/using-the-network-shim.md

```diff
@@ -149,42 +135,72 @@ This file is meant to be git-ignored and we can use this to set local environmen
 
 _Note that `dhis2Username` and `dhis2Password` are not required directly by the Network Shim, but by the `enableAutologin` setup. They are included in this example since they will be needed for most use-cases._
 
-#### Plugins file
+#### Plugin setup
 
-The `networkShim` plugin should be imported and initialised in the cypress plugins file, see the [cypress docs](https://docs.cypress.io/guides/tooling/plugins-guide#Using-a-plugin) for more info. The plugin function doesn't require the second `config` argument most plugins accept.
+The `networkShim` plugin should be imported and initialised in the
+setupNodeEvents function in the cypress config file (for more information about
+the config file, see [Getting started](../getting-started.md)):
 
 ```js
-const { networkShim } = require('@dhis2/cypress-plugins')
+import { networkShim } from '@dhis2/cypress-plugins'
 
-module.exports = (on, config) => {
-    networkShim(on)
+// ...
+
+async function setupNodeEvents(on, config) {
+    // ...
+
+    networkShim(on, config)
+
+    // ...
 }
+
+// ...
 ```
 
-The `networkShim` plugin accepts a second `options` argument. For most DHIS2 platform apps the default options will be correct so there will be no need to pass a second argument.
+The `networkShim` plugin accepts a second `options` argument. For most DHIS2
+platform apps the options passed to `setupNodeEvents` will be correct so there
+will be no need to pass a second argument.
 
-For most DHIS2 Cypress test suites the `enableAutoLogin` plugin can help reduce repetitive login steps in tests. In turn, the `enableAutoLogin` setup utility requires the `chromeAllowXSiteCookies` plugin when using Chrome. So a typical DHIS2 app will use this plugin alongside the `networkShim`.
+For most DHIS2 Cypress test suites the `enableAutoLogin` plugin can help reduce
+repetitive login steps in tests. In turn, the `enableAutoLogin` setup utility
+requires the `chromeAllowXSiteCookies` plugin when using Chrome. So a typical
```

Comment:
I think we should look into the value this plugin still has. I have some vague recollection of some Chrome updating breaking this. Have also found a [StackOverflow post](https://stackoverflow.com/questions/69150082/how-can-i-disable-same-site-cookies-flags-after-latest-update-of-chrome93) which seems to confirm this vague recollection.

Reply:
We talked about it and agreed to keep it as it doesn't hurt and won't ever change (no maintenance required) as Chrome/Chromium v94+ does not support this anymore.

I'm still using it as I have the latest Chrome but Chromium v93 installed on purpose as the developer tools are nicer to use than electron's dev tools.